DomainModels/Order.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text.Json.Serialization;
+
+namespace DomainModels
+{
+    public class Order : IEntity
+    {
+        public long Id { get; set; }
+        public int Quotes { get; set; }
+        public decimal UnitPrice { get; set; }
+        public decimal NetValue { get; set; }
+        public DateTime LiquidatedAt { get; set; }
+        public OrderDirection Direction { get; set; }
+        public Portfolio Portfolio { get; set; }
+        public long PortfolioId { get; set; }
+        public Product Product { get; set; }
+        public long ProductId { get; set; }
```

Comment:
Dica de organização e visibilidade, fique a vontade pare replicar nas demais
```suggestion

        public long PortfolioId { get; set; }
        public Portfolio? Portfolio { get; set; }

        public long ProductId { get; set; }
        public Product? Product { get; set; }
```